gilbreth_perception/launch/gilbreth_perception.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+
+<group>
+    <node pkg="gilbreth_perception" name="tf_publisher" type="tf_publisher" args="0.70711 -0.70711 0 0"/>
+  </group>
+
+  
+  <group>
```

Comment:
The <group> should be used if you're namespacing several nodes and parameters or if you are trying to run things conditionally `if=$(arg sim)` ,  In this case there isn't much need for the group tag.